build/templates/template-bot-resources-legacy.json

```diff
@@ -0,0 +1,249 @@
+{
+    "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion":"1.0.0.0",
+    "parameters":{
+       "groupLocation":{
+          "type":"string",
+          "metadata":{
+             "description":"Specifies the location of the Resource Group."
+          }
+       },
+       "groupName":{
+          "type":"string",
+          "metadata":{
+             "description":"Specifies the name of the Resource Group."
+          }
+       },
+       "appId":{
+          "type":"string",
+          "metadata":{
+             "description":"Active Directory App ID, set as MicrosoftAppId in the Web App's Application Settings."
+          }
+       },
+       "appSecret":{
+          "type":"string",
+          "metadata":{
+             "description":"Active Directory App Password, set as MicrosoftAppPassword in the Web App's Application Settings."
+          }
+       },
+       "botId":{
```

Comment:
Why are the new templates named "legacy"?

Reply:
Hi @mrivera-ms for the name 'legacy', Gabo and us decided to call the current pipelines 'legacy' since they are meant to be replaced with the new ones eventually.  

As for the existence, these new templates were introduced for the legacy pipelines, so they keep working in exchange for the removed template files.